lib/internal/http2/compat.js

```diff
@@ -416,8 +416,18 @@ class Http2ServerResponse extends Stream {
 
   write(chunk, encoding, cb) {
     var stream = this[kStream];
+
+    if (typeof encoding === 'function') {
```

Comment:
`chunk` is required: https://nodejs.org/api/stream.html#stream_writable_write_chunk_encoding_callback